### PR TITLE
Added PR template and publish action

### LINKED ISSUE #285: Add PR template and update publish action
@AbhayVAshokan _a

### PR DESCRIPTION
**Pull request by AbhayVAshokan**
Fixes #285 

- Added PR template.
- Synced CHANGELOG.md file with neetoUI.
- Synced `.github/workflows/publish.yml` workflows with neetoUI.

@amaldinesh7 _a please have a look. The PR template mentions about adding necessary labels. This practice has not been followed in neetoEditor. Hence, do check if the necessary changes for the same are added. I am not sure what the functionality of `enforce-label.yml` file is, so haven't added the same.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,10 @@
 
 ------ FOLLOW THESE WHILE ADDING AN ENTRY ------
 
-** Add BREAKING keyword in bold for changes which could potentially break the component, eg: **BREAKING**
-** Enclose a prop name in double backticks, eg: `isLoading`
+** Add BREAKING keyword in bold for changes which could potentially break the component, eg: **BREAKING**.
+** Represent a component name in italics, eg: _Modal_.
```
Comment (amaldinesh7) [inline CHANGELOG.md:6]: Remove this

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,8 +2,10 @@
...
+** Enclose a prop name in double backticks, eg: `isLoading`.
```
Comment (amaldinesh7) [inline CHANGELOG.md:7]: change the `isLoading` prop to some prop in neetoEditor. Do the same in PR template file as well.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+- Fixed: for any bug fixes.
+- Security: in case of vulnerabilities.
+
+Points to note:
+- The description shall be represented in bullet points.
+- Add the keyword BREAKING in bold style for changes that could potentially break the component, eg: **BREAKING**.
+- Represent a component name in italics, eg: _Modal_.
+- Enclose a prop name in double backticks, eg: `isLoading`.
+
+Example:
```
Comment (amaldinesh7) [inline .github/pull_request_template.md:34]: Remove this for now.

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]
@AbhayVAshokan Please check the comments.

**Comment by amaldinesh7** [comment]
I've created the necessary labels. The logic of enforce-label is it will add `skip-ci` label if the changes in the PR need not be published. For example if a PR only have changes to update documentation, we don't have to publish a new version.

**Comment by AbhayVAshokan** [comment]
@amaldinesh7 _a

**Review by amaldinesh7 — APPROVED** [review]

**Pull request merged**